FBC/DebtAndEquities/Debt.rdf

```diff
@@ -208,6 +208,8 @@
 							</rdf:Description>
 							<rdf:Description rdf:about="&fibo-fbc-fi-fi;Security">
 							</rdf:Description>
+							<rdf:Description rdf:about="&fibo-fnd-agr-ctr;PromissoryNote">
```

Comment:
So - this is the collateralizationOf property, which refers to the instrument rather than the asset.  I think we just say Asset as the thing that is used as collateral at this point.

Reply:
I agree